src/emu/machine.cpp

```diff
@@ -1322,16 +1329,13 @@ static running_machine * jsmess_machine;
 
 void js_main_loop()
```

Comment:
Is it necessary for this to be a free function?  It seems like more of a case for a static member function of the `running_machine`.